.github/workflows/README.md

```diff
@@ -0,0 +1,55 @@
+# GitHub Actions Workflows
+
+## Setup
+- workloads run using [GitHub self-hosted runners](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-self-hosted-runners)
+- project admins maintain a private Google Compute Engine VM for running tests
+  - VM should be at least n1-standard-4 with 50GB persistent disk
+  - instructions for setting up the VM can be found in repo settings under "Actions"
+  - to set up dependencies, run the following commands:
+    ```
+    # install kubectl
+    sudo apt-get install kind kubectl
```

Comment:
which one installs kind? this or below?

Reply:
good catch, it's the second one. I'll remove this